impress-org/give-double-the-donation
# Feature: Add selected company to confirmation page

## Description

[GIVE-493]

This PR adds company matching instructions on the Confirmation page.

## Affects

Confirmation page

## Visuals


![Screen Capture_select-area_20240325082838](https://github.com/impress-org/give-double-the-donation/assets/4222590/eea3a523-245d-4c2c-b4f2-bf18fef57056)


## Testing Instructions

Create a form with Double the Donation block
View form on the frontend
Select Company
Donate

The matching instructions should be visible at the bottom of the confirmation page


## Pre-review Checklist






[GIVE-493]: https://stellarwp.atlassian.net/browse/GIVE-493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
